crates/flux-fhir-analysis/src/conv/mod.rs

```diff
@@ -2022,6 +2022,7 @@ fn conv_lit(lit: fhir::Lit) -> rty::Constant {
         fhir::Lit::Real(r) => rty::Constant::Real(rty::Real(r)),
         fhir::Lit::Bool(b) => rty::Constant::from(b),
         fhir::Lit::Str(s) => rty::Constant::from(s),
+        fhir::Lit::Char(s) => rty::Constant::from(s as i128),
```

Comment:
```suggestion
        fhir::Lit::Char(c) => rty::Constant::Char(c),
```

or you can define `impl From<char> for Constant`